api/operations.py

```diff
@@ -109,17 +109,25 @@ async def create_operation(
 
         Both modes are secure - the blockchain permission defines the security boundary.
 
+    Replay Protection:
+        All requests must include a timestamp field for replay attack protection:
+        - timestamp (required): Unix timestamp for request freshness validation
+        - Requests must be within 15 minutes of server time (configurable)
+        - Prevents captured signatures from being reused days/weeks later
+        - Based on AWS Signature V4 security model
+
     Request Format:
         The operation_request_json field (which is signed) can include:
         - permission_id (required): Blockchain permission ID
+        - timestamp (required): Unix timestamp for replay protection
```

Comment:
I don't see where is the signature generated, make sure that the `timestamp` is included in the signing message.